Making rint static breaks VxWorks builds

Commit e3ec0a9042414144fc555348b7e11b233e3c3c15 breaks builds on VxWorks:
```
../smarActMCSMotorDriver.cpp: In function 'double rint(double)':
../smarActMCSMotorDriver.cpp:41: error: 'double rint(double)' was declared 'extern' and later 'static'
/APSshare/vw/vxWorks-6.9.4/vxworks-6.9.4/target/h/math.h:104: error: previous declaration of 'double rint(double)'
make[5]: *** [smarActMCSMotorDriver.o] Error 1
make[5]: Leaving directory `/net/s100dserv/xorApps/epics/motor-devel/support/motor-git/modules/motorSmarAct/smarActApp/src/O.vxWorks-ppc32'
make[4]: *** [install.vxWorks-ppc32] Error 2
make[4]: Leaving directory `/net/s100dserv/xorApps/epics/motor-devel/support/motor-git/modules/motorSmarAct/smarActApp/src'
make[3]: *** [src.install] Error 2
make[3]: Leaving directory `/net/s100dserv/xorApps/epics/motor-devel/support/motor-git/modules/motorSmarAct/smarActApp'
make[2]: *** [smarActApp.install] Error 2
make[2]: Leaving directory `/net/s100dserv/xorApps/epics/motor-devel/support/motor-git/modules/motorSmarAct'
make[1]: *** [motorSmarAct.install] Error 2
make[1]: Leaving directory `/net/s100dserv/xorApps/epics/motor-devel/support/motor-git/modules'
make: *** [modules.install] Error 2
```

--- a/smarActApp/src/smarActMCSMotorDriver.cpp
+++ b/smarActApp/src/smarActMCSMotorDriver.cpp
@@ -35,9 +35,9 @@
 #define FAR_AWAY     1000000000 /*nm*/
 #define UDEG_PER_REV 360000000
 
-// Windows does not have rint(), but minGW does
-#if defined __MINGW32__ || defined __MINGW64__
-#elif defined _WIN32
+#ifdef __MSC__
+/* MSC may not have rint() function */
+#if(_MSC_VER < 1900)
 static double rint(double x)
 {
   //middle value point test
@@ -50,7 +50,7 @@ static double rint(double x)
   else return floor(x+0.5);
 }
 #endif
-  
+#endif
 
 /* The asyn motor driver apparently can't cope with exceptions */
 #undef  ASYN_CANDO_EXCEPTIONS

--- a/smarActApp/src/smarActMCSMotorDriver.cpp
+++ b/smarActApp/src/smarActMCSMotorDriver.cpp
@@ -35,9 +35,9 @@
 #define FAR_AWAY     1000000000 /*nm*/
 #define UDEG_PER_REV 360000000
 
-// Windows and vxWorks do not have rint(), but minGW does
+// Windows does not have rint(), but minGW does
 #if defined __MINGW32__ || defined __MINGW64__
-#elif defined _WIN32 || defined vxWorks
+#elif defined _WIN32
 static double rint(double x)
 {
   //middle value point test

--- a/smarActApp/src/smarActSCUMotorDriver.cpp
+++ b/smarActApp/src/smarActSCUMotorDriver.cpp
@@ -36,9 +36,9 @@
 #define UDEG_PER_REV 360000000
 #define STEPS_PER_EGU 1000.
 
-// Windows and vxWorks do not have rint(), but minGW does
+// Windows does not have rint(), but minGW does
 #if defined __MINGW32__ || defined __MINGW64__
-#elif defined _WIN32 || defined vxWorks
+#elif defined _WIN32 
 static double rint(double x)
 {
   //middle value point test

--- a/smarActApp/src/smarActSCUMotorDriver.cpp
+++ b/smarActApp/src/smarActSCUMotorDriver.cpp
@@ -36,23 +36,6 @@
 #define UDEG_PER_REV 360000000
 #define STEPS_PER_EGU 1000.
 
-// Windows does not have rint(), but minGW does
-#if defined __MINGW32__ || defined __MINGW64__
-#elif defined _WIN32 
-static double rint(double x)
-{
-  //middle value point test
-  if (ceil(x+0.5) == floor(x+0.5))
-  {
-    int a = (int)ceil(x);
-    if (a%2 == 0) return ceil(x);
-    else return floor(x);
-  }
-  else return floor(x+0.5);
-}
-#endif
-  
-
 /* The asyn motor driver apparently can't cope with exceptions */
 #undef  ASYN_CANDO_EXCEPTIONS
 /* Define this if exceptions should be thrown and it is OK to abort the application */

